docs/2-benchpark-list.rst

```diff
@@ -1,15 +1,15 @@
 ===================
-Benchpark workspace
+Benchpark list
 ===================
+To list all available in Benchpark benchmarks and systems::
 
-Benchpark experiments are organized as a 
-`Ramble workspace <https://googlecloudplatform.github.io/ramble/workspace.html>`_, 
-a set of self-contained configuration files required 
-to correctly build and execute those experiments. 
+  benchpark list
  
 Configuration files are organized as follows:: 
 
   $benchpark 
+  ├── bin
+  │  ├── benchpark
   ├── configs 
   │  ├── ${SYSTEM1} 
   │  │  ├── auxiliary_software_files 
```

Comment:
This listing is repeated in https://github.com/LLNL/benchpark/pull/24/files#diff-dcd35bb8e1b7a8850b7180d78da82906aac0c9703c78aa45b4d73c92c7aa7d81 - I think it might as well be deleted here if we're just talking about out to list out available systems/benchmarks (I don't think the user needs to know the config file layout to select/run benchmarks)

I'd say the same for everything after this as well.